src/main/resources/migration/common/V11__create_licence_tables.sql

```diff
@@ -41,8 +41,7 @@ CREATE TABLE licence (
   probation_area_code varchar(10),
   probation_ldu_code varchar(20),
   appointment_person varchar(60),
-  appointment_date date,
-  appointment_time varchar(8),
+  appointment_time timestamp with time zone,
```

Comment:
Is Flyway going to complain at this?

Reply:
Sorry just saw your comment on the PR - going to replace the DEV database. A bit of history rewriting there :smile:

Reply:
Keeps thing tidy - we're only in DEV at the moment. This luxury will go away soon. Thanks